### PR TITLE
Allow more maintenance

### PR DESCRIPTION
**Pull request by teamsolidus**

### CODE REVIEW

**Comment by morxa** [comment]
In my opinion, this will result in too many maintenances. Up to six maintenances per robot with six robots in the game can result in up to 36 maintenances, that's a lot.


_If_ we want to allow more maintenances, I'd favor something stricter, e.g., if there is no activity by either team, both teams are allowed to do additional maintenance. This would avoid the problem of games where nothing happens anymore at the end of the game, but at the same time would give the teams a high incentive to keep their robots running.

**Comment by teamsolidus** [comment]
> In my opinion, this will result in too many maintenances. Up to six maintenances per robot with six robots in the game can result in up to 36 maintenances, that's a lot.
> 
> _If_ we want to allow more maintenances, I'd favor something stricter, e.g., if there is no activity by either team, both teams are allowed to do additional maintenance. This would avoid the problem of games where nothing happens anymore at the end of the game, but at the same time would give the teams a high incentive to keep their robots running.

**Comment by timn** [comment]
Why has this been closed? Was it by accident with "Close and comment" or intentional?

**Comment by teamsolidus** [comment]
i didn't want to close it, i can't handle this tool correctly

**Comment by morxa** [comment]
I rebased the branch onto the latest master.

File: rulebook.tex [hunk 1]
```diff
@@ -1295,12 +1303,6 @@ \subsection{Interruptions and Robot Maintenance}
 so rarely as not to interfere with the overall game flow (also
 cf.~\refsec{sec:during-match}).
 
-If a robot needs to be taken out for the second time, either on
```
Comment (timn) [inline rulebook.tex]: We still need this part, only that this applies to the third maintenance now.

Comment (thomerz) [inline rulebook.tex]: Thanks for pointing that out. I somehow accidentally deleted that paragraph. Should be re-inserted now. It also states that the robot is now disqualified if it needs to be taken out for the third time, in order to account for the now allowed second maintenance.

**Comment by timn** [comment]
Looks good to me.

**Comment by morxa** [comment]
This seems to be ready. Please leave your vote!

**Comment by timn** [comment]
+1

**Comment by teamsolidus** [comment]
+1

**Comment by morxa** [comment]
+1

**Comment by thomerz** [comment]
+1

**Comment by morxa** [comment]
**Proposal accepted**

* in favor: 4
* abstentions: 3
* opposed: 0

(Closing early, as a majority voted in favor).

**Comment by MostafaGomaa** [comment]
+1

**Comment by MostafaGomaa** [comment]
oh :)..nice timing

**Pull request merged**